src/main/offline/io/stargate/sgv2/jsonapi/service/cqldriver/sstablewriter/OfflineFileWriterInitializer.java

```diff
@@ -0,0 +1,27 @@
+package io.stargate.sgv2.jsonapi.service.cqldriver.sstablewriter;
+
+import static io.stargate.sgv2.jsonapi.service.cqldriver.CQLSessionCache.OFFLINE_WRITER;
+
+import io.smallrye.config.SmallRyeConfig;
+import io.smallrye.config.SmallRyeConfigBuilder;
+import io.smallrye.context.SmallRyeContextManagerProvider;
+import io.stargate.sgv2.jsonapi.config.OperationsConfig;
+import io.stargate.sgv2.jsonapi.config.constants.ApiConstants;
+
+public class OfflineFileWriterInitializer {
+
+  public static void initialize() {
+    System.setProperty(ApiConstants.OFFLINE_WRITER_MODE_PROPERTY, "true");
+    SmallRyeContextManagerProvider.instance();
```

Comment:
Why is this needed?

Reply:
This is needed one time in the startup for the SmallRye config parsing to work in the subsequent flows